### PR TITLE
Adds Sequencer OpenAPI spec

### PR DESCRIPTION
**Pull request by dylanlott**
- Updates documentation for sequencer API.
- Updates README to reflect new environment variables

### CODE REVIEW

**Review by rswanson — APPROVED** [review]

**Review by prestwich — CHANGES_REQUESTED** [review]
types are not correct as U256 serializes to string not number

you should invoke the json serializer and document the output

**Comment by prestwich** [comment]
most of these could use additional notation that they are formatted as Ethereum `Quantity`, which is to say, a 0x-prefixed minimal hex string

File: crates/builder/sequencer.openapi.yml [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+      - host_block_number
+      - host_chain_id
+      - ru_chain_id
+      - gas_limit
+      - ru_reward_address
+      - contents
+    Signature:
+      type: object
+      properties:
+        v:
```
Comment (prestwich) [inline crates/builder/sequencer.openapi.yml:121]: i believe this is actually incorrect. The V value is serialized based on parity rules. This _should_ be a boolean called `yParity`, and if we want to guarantee that, we can do so in quincey

**Review by prestwich — APPROVED** [review]

**Comment by dylanlott** [comment]
### Merge activity

* **Aug 28, 12:59 PM EDT**: @dylanlott started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/init4tech/zenith-rs/38).

**Pull request merged**